openshift/template.yaml

```diff
@@ -0,0 +1,125 @@
+apiVersion: v1
+kind: Template
+labels:
+  template: 3scale-admin-gateway
+metadata:
+  name: 3scale-admin-gateway
+  annotations:
+    description: 3scale-admin-gateway
+objects:
+- apiVersion: v1
+  kind: DeploymentConfig
+  metadata:
+    labels:
+      service: 3scale-admin-gateway
+    name: 3scale-admin-gateway
+  spec:
+    selector:
+      service: 3scale-admin-gateway
+    template:
+      metadata:
+        labels:
+          service: 3scale-admin-gateway
+      spec:
+        containers:
+        - env:
+          - name: F8A_DEBUG
+            value: "true"
+          - name: DEPLOYMENT_PREFIX
+            valueFrom:
+              configMapKeyRef:
+                name: bayesian-config
+                key: deployment-prefix
+          - name: BAYESIAN_FETCH_PUBLIC_KEY
+            valueFrom:
+              configMapKeyRef:
+                name: bayesian-config
+                key: keycloak-url
+          - name: BAYESIAN_JWT_AUDIENCE
+            value: "fabric8-online-platform,openshiftio-public"
+          image: "${DOCKER_REGISTRY}/${DOCKER_IMAGE}:${IMAGE_TAG}"
+          name: 3scale-admin-gateway
+          ports:
+          - containerPort: 5000
+            protocol: TCP
+          readinessProbe:
+            httpGet:
+              path: /api/v1/readiness
```

Comment:
I think there is no such endpoint, the same for liveness probe. So the container will never be ready.

Reply:
Does something extra need to be handled here?

Reply:
There is no such endpoint in your application so your container will be crash-looping. I guess you could replace it with `/`. `index.html` page should do the trick here.